CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+*                                         @ridedott/platform
+
+.github/ISSUE_TEMPLATE                    @ridedott/platform
+.github/pull_request_template.md          @ridedott/platform
+
+.vscode                                   @ridedott/platform
+
+decisions/DECISION_TEMPLATE.md            @ridedott/platform
+decisions/README.md                       @ridedott/platform
+
+
+.editorconfig                             @ridedott/platform
+
+.prettierignore                           @ridedott/platform
+.prettierrc.yml                           @ridedott/platform
+
+CODEOWNERS                                @ridedott/platform
+
+commitlint.config.js                      @ridedott/platform
+CONTRIBUTING.md                           @ridedott/platform
+
+tsconfig.json                             @ridedott/platform
+tsconfig.production.json                  @ridedott/platform
```

Comment:
Couldn't we just keep the top line and have other owners added if needed?

Reply:
Yeah we only need the wildcard line. Everything else, including the comments can go. It's self explanatory enough and documentation is easy to find.